WooCommerce/Classes/ViewRelated/Products/Variations/ProductVariationsTopBannerFactory.swift

```diff
@@ -0,0 +1,25 @@
+import UIKit
+
+/// Generates top banner view that is shown at the top of variation list screen when at least one variation is missing a price.
+///
+final class ProductVariationsTopBannerFactory {
+    static func missingPricesTopBannerView() -> TopBannerView {
+        let viewModel = TopBannerViewModel(title: Constants.title,
+                                           infoText: Constants.info,
+                                           icon: Constants.icon,
+                                           isExpanded: true,
+                                           topButton: .none,
+                                           type: .warning)
+        return TopBannerView(viewModel: viewModel)
+    }
+}
+
+private extension ProductVariationsTopBannerFactory {
+    enum Constants {
```

Comment:
nit: maybe it's better to call it `Localization` or `Strings` and move the icon under `enum Constants`

Reply:
updated in https://github.com/woocommerce/woocommerce-ios/pull/2664/commits/8e00aacf957c9b3397edc91b44211fe9b852f1bc